modules/bluetooth.sh

```diff
@@ -5,100 +5,111 @@ function bluetooth {
   checkargn $# 2
   status=$1
 
-  if [ -z "$status" ]; then
-    if [[ "$(service rpibluetooth status | grep "Active:")" =~ "running" ]]; then
-      echo "on"
-    else
-      echo "off"
-    fi
+  case $status in
+    -z)
```

Comment:
wrong

Reply:
@sarahrams how can you use a string to represent `-z`?

Reply:
That was a mistake. I already updated.